base_layer/core/Cargo.toml

```diff
@@ -5,6 +5,10 @@ edition = "2018"
 
 [dependencies]
 bitflags = "1.0.4"
+byteorder = "1.3.1"
 chrono = "0.4.6"
 digest = "0.8.0"
-crypto = { path = "../../infrastructure/crypto"}
\ No newline at end of file
+crypto = { path = "../../infrastructure/crypto"}
+curve25519-dalek = "1.0.2"
```

Comment:
What parts of dalek are you using here? We should try make it such that we only depend on the `crypto` crate; so if we change out the crypto library (incl version upgrades) there's one place that happens

Reply:
Maybe you're just waiting for #137 and #63 to be done, and then this can go away.

Reply:
I am only using `Scalar` because I need to be able to create `HomomorphicCommitments` which uses parameters of that type. Perhaps we will get rid of this when we refactor `HomomorphicCommitments`.